frontend/packages/data-portal/app/graphql/getRunByIdDiffer.ts

```diff
@@ -54,6 +54,41 @@ export function logIfHasDiff(
     // There are no frames in V1.
     delete run.framesAggregate
   }
+  // Don't care about counts.
+  for (const aggregate of v2.uniqueAnnotationSoftwares.aggregate ?? []) {
+    delete aggregate.count
+  }
+  for (const aggregate of v2.uniqueObjectNames.aggregate ?? []) {
+    delete aggregate.count
+  }
+  for (const aggregate of v2.uniqueShapeTypes.aggregate ?? []) {
+    delete aggregate.count
+  }
+  for (const aggregate of v2.uniqueResolutions.aggregate ?? []) {
+    delete aggregate.count
+  }
+  for (const aggregate of v2.uniqueProcessingMethods.aggregate ?? []) {
+    delete aggregate.count
+  }
+  // Consistent sort order.
+  v2.uniqueAnnotationSoftwares.aggregate?.sort((gropuByA, groupByB) =>
+    gropuByA.groupBy!.annotationSoftware!.localeCompare(
+      groupByB.groupBy!.annotationSoftware!,
+    ),
+  )
+  v2.uniqueObjectNames.aggregate?.sort((groupA, groupB) =>
+    groupA.groupBy!.objectName!.localeCompare(groupB.groupBy!.objectName!),
+  )
+  v2.uniqueShapeTypes.aggregate?.sort((groupA, groupB) =>
+    groupA.groupBy!.shapeType!.localeCompare(groupB.groupBy!.shapeType!),
+  )
+  v2.uniqueResolutions.aggregate?.sort(
+    (groupA, groupB) =>
+      groupA.groupBy!.voxelSpacing! - groupB.groupBy!.voxelSpacing!,
+  )
+  v2.uniqueProcessingMethods.aggregate?.sort((groupA, groupB) =>
+    groupA.groupBy!.processing!.localeCompare(groupB.groupBy!.processing!),
+  )
```

Comment:
Good callout, so yes, but it shouldn't have to be, it's possible to configure GQL codegen to make these required since we're querying them. Logged an issue here https://github.com/chanzuckerberg/cryoet-data-portal/issues/1426